app/assets/javascripts/pageflow/editor/views/change_theme_view.js

```diff
@@ -0,0 +1,35 @@
+pageflow.ChangeThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'templates/change_theme',
+
+  ui: {
+    changeThemeButton: '.change_theme',
+    labelText: 'label .name'
+  },
+
+  events: {
+    'click .change_theme': function() {
+      pageflow.ChangeThemeDialogView.open({
+        model: this.model,
+        themes: this.options.themes
+      });
+    }
+  },
+
+  onRender: function() {
+    this.ui.labelText.text(this.labelText());
+  },
+
+  labelText: function() {
+    return this.options.label || this.labelPrefix() + ': ' + this.localizedThemeName();
```

Comment:
That would be a duplication since he theme selection dialog already takes the role of that select input. I'd just put the name of the current theme below the label as plain text, much like the name of the current file is currently displayed in file input views. Then the user can click the button to open the dialog and make changes.